### ISSUE TITLE
flynn-host setup hangs while trying to connect to 1113 (following reboot)

### ISSUE THREAD

**Issue by adamcharnock** [flynn/flynn#2608]
I just rebooted my flynn instance (chmod was consuming 100% cpu and grinding the machine to a halt). However, now flynn seems to be hanging while starting. Running 'flynn-host ps' just hangs. I see this in my logs:

```
233 <0>1 2016-03-20T23:55:52.817773072Z ip1000100 b180ad4a-53c9-495c-a612-7464740368b7 app.ip1000100-ac358e5b-c84f-4370-851d-6e6fc938b65d ID2 [flynn seq="33"] 2016/03/20 23:55:52 desired host status still not reached after 1m0.029601876s
```

Going by [the source](https://github.com/flynn/flynn/blob/ec52752f508536c8a749bd3a071b2af7296c9b85/pkg/cluster/host.go#L84) it seems that it is trying to connect to something on 1113. So, perhaps something involving discoverd (which is running)?

I'd post the `collect-debug-info` details, but I think that command is hanging too.

**Comment by titanous** [comment]
We'll need the logs from `/var/log/flynn` and `/var/log/upstart/flynn-host.log`.

**Comment by adamcharnock** [comment]
@titanous Here you go: https://dl.dropboxusercontent.com/u/41936/logs.tar.gz

**Comment by titanous** [comment]
@adamcharnock It looks like the flynn-host log was rotated, are there any older versions of the log in `/var/log/upstart`?

**Comment by titanous** [comment]
If not, restart the flynn-host process and then run `sudo flynn-host collect-debug-info` after five minutes or so.

**Comment by adamcharnock** [comment]
Sorry for the delay @titanous: All flynn-host logs from ``var/log/upstart`` are here: https://dl.dropboxusercontent.com/u/41936/flynn-host-all-logs.tar.gz

Following downloading those logs, I've restarted flynn-host. However, ``sudo flynn-host collect-debug-info`` still seems to hang.

**Comment by titanous** [comment]
This looks like the deadlock fixed in #2591.

**Comment by titanous** [comment]
The fix for this is in the current stable release.
